src/devices/Ad7193/Channel.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+
+namespace Iot.Device.Ad7193
+{
+    /// <summary>
+    /// Channel selection flags
+    /// </summary>
+    [Flags]
+    public enum Channel
+    {
+        /// <summary>
+        /// Channel CH0
+        /// (Pseudo Bit = 0) -> +:AIN1 and -:AIN2
+        /// (Pseudo Bit = 1) -> +:AIN1 and -:AINCOM
+        /// </summary>
+        CH00 = 0b00_0000_0001,
```

Comment:
Channel0, Channel1 etc.. also for bits I'd be easier to read if you used `1 << 0`, `1 << 1`, `1 << 2`, ...